src/runtime/vdom/vdom-render.ts

```diff
@@ -187,6 +187,13 @@ const putBackInOriginalLocation = (parentElm: Node, recursive: boolean) => {
       // Reset so we can correctly move the node around again.
       childNode['s-sh'] = undefined;
 
+      // When putting an element node back in it's original location,
+      // we need to reset the `slot` attribute back to the value it originally had
+      // so we can correctly relocate it again in the future
+      if (childNode.nodeType === NODE_TYPE.ElementNode && !childNode['s-sr']) {
```

Comment:
For my own edification, why do we check that `childNode['s-sr']` is falsy?

Reply:
Actually, good catch. We don't need to since a slot ref node can only be a text node or comment node, so they'd never pass the first check anyway

Reply:
Wooo!